pkg/storage/buckets/store.go

```diff
@@ -222,3 +248,69 @@ func convertMeta(ctx context.Context, m objects.Meta) (out Meta, err error) {
 
 	return out, err
 }
+
+// convertTime converts gRPC timestamp to Go time
+func convertTime(ts *timestamp.Timestamp) time.Time {
```

Comment:
I'm not entirely sure, but I think you might be able to avoid the need for this entirely by using a protobuf extension: `(gogoproto.stdtime) = true`. See pkg/pb/gc.proto for usage.